django/contrib/admin/options.py

```diff
@@ -1944,7 +1944,22 @@ def _create_formsets(self, request, obj, change):
                     'files': request.FILES,
                     'save_as_new': '_saveasnew' in request.POST
                 })
-            formsets.append(FormSet(**formset_params))
+            formset = FormSet(**formset_params)
+
+            def _should_delete(request, obj, formset, index):
+                """If we have delete permission, check for DELETE key in form"""
```

Comment:
Avoid use of "we" in comments per [style guide](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style). "Return whether or not the user deleted the form."